libusb/os/windows_winusb.c

```diff
@@ -724,6 +725,50 @@ static void winusb_exit(struct libusb_context *ctx)
 	exit_dlls();
 }
 
+static BOOL device_io_control(HANDLE hDevice, DWORD dwIoControlCode, LPVOID lpInBuffer, DWORD nInBufferSize, LPVOID lpOutBuffer, DWORD nOutBufferSize, LPDWORD lpBytesReturned) 
+{
+	OVERLAPPED overlapped;
+	HANDLE io_event;
+	DWORD error;
+	BOOL ret = TRUE;
+
+	io_event = CreateEvent(NULL, FALSE, FALSE, NULL);
```

Comment:
creating an event is not a cheap operation
it should be cached
create it once and cache along with the device handle

in fact, you can cache the entire `overlapped`